samples/bluetooth/mesh/sensor_client/README.rst

```diff
@@ -0,0 +1,182 @@
+
+User interface
+**************
+
+Buttons:
+   Can be used to input the OOB authentication value during provisioning.
+   All buttons have the same functionality during this procedure.
+
+Terminal:
+   All sensor values gathered from the server are printed over UART.
+   For more details, see :ref:`gs_testing`.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/sensor_client`
+
+.. include:: /includes/build_and_run.txt
+
+.. _bluetooth_mesh_sensor_client_testing:
+
+Testing
+=======
+
+.. note::
+   The Mesh sensor client sample cannot demonstrate any functionality on its own, and needs a device with the :ref:`bluetooth_mesh_sensor_server` sample running in the same mesh network.
+   Before testing the Mesh sensor client, go through the Mesh sensor server's :ref:`testing guide <bluetooth_mesh_sensor_server_testing>` with a different board.
+
+After programming the sample to your board, you can test it by using a smartphone with Nordic Semiconductor’s nRF Mesh app installed.
+Testing consists of provisioning the device and configuring it for communication with the mesh models.
+
+All sensor values gathered from the server are printed over UART.
+For more details, see :ref:`gs_testing`.
+
+Provisioning the device
+-----------------------
+
+The provisioning assigns an address range to the device, and adds it to the mesh network.
+Complete the following steps in the nRF Mesh app:
+
+1. Tap :guilabel:`Add node` to start scanning for unprovisioned mesh devices.
+#. Select the :guilabel:`Mesh Sensor Client` device to connect to it.
+#. Tap :guilabel:`Identify` and then :guilabel:`Provision` to provision the device.
+#. When prompted, select an OOB method and follow the instructions in the app.
+
+Once the provisioning is complete, the app returns to the Network screen.
+
+Configuring models
+------------------
+
+Complete the following steps in the nRF Mesh app to configure models:
+
+1. On the Network screen, tap the :guilabel:`Mesh Sensor Client` node.
+   Basic information about the mesh node and its configuration is displayed.
+#. In the Mesh node view, expand the element.
+   It contains the list of models in the first and only element of the node.
+#. Tap :guilabel:`Sensor Client` to see the model's configuration.
+#. Bind the model to application keys to make it open for communication:
+
+   a. Tap :guilabel:`BIND KEY` at the top of the screen.
+   #. Select :guilabel:`Application Key 1` from the list.
+
+#. Set the publishing parameters:
+
+   a. Tap :guilabel:`SET PUBLICATION`.
+   #. Tap :guilabel:`Publish Address`.
+   #. Select :guilabel:`Groups` from the drop-down menu.
+   #. Select an existing group or create a new one.
+
+      .. note::
+        The sensor server must subscribe to the same group.
+
+   #. Tap :guilabel:`OK`.
+   #. Set the Retransmit Count to zero (:guilabel:`Disabled`) to avoid duplicate logging in the UART terminal.
```

Comment:
Modified this in both .rst files, but please double check that Disabled matches the app.

Reply:
Disabled does match the app

Reply:
OK